Makefile

```diff
@@ -38,6 +58,9 @@ latest:
 	# (run by CI only for releases)
 	docker push $(REPO):latest
 
+check:
```

Comment:
This could be changed to include the code necessary to install.  Something like
```
bin/golangci-lint:
     curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./bin v1.27.0

check: bin/golangci-lint
    ./bin/golangci-lint run --enable=unconvert,prealloc,bodyclose
```
This would enable `make check` to be run by a developer (on a linux host at least).